README.md

```diff
@@ -1,4 +1,6 @@
-# Keptn-Sockshop/Carts
+:warning: The 0.13.0 release was the last release for this repository since the Keptn team moves to another application for demo purposes of Keptn. No maintenance of this repository anymore.
```

Comment:
```suggestion
:warning: The 0.13.0 release was the last release for this repository since the Keptn team moves to another application for demo purposes of Keptn. No further maintenance will be provided on this repository.
```